web/beacon-app/src/features/topics/components/QueryInput.tsx

```diff
@@ -0,0 +1,29 @@
+import { Trans } from '@lingui/macro';
+import { Button, TextField } from '@rotational/beacon-core';
+
+type TopicNameProps = {
+  name: string;
+};
+
+const QueryInput = ({ name }: TopicNameProps) => {
+  return (
+    <div className="mt-4 flex space-x-2">
+      <TextField
+        labelClassName="Topic Query"
+        type="search"
+        value={`SELECT * FROM ${name} LIMIT 1`}
```

Comment:
Using `defaultValue` returns an `undefined` name here. Using `useState` also returned `undefined`. Since this is for the view, I decided to handle this issue when working on implementing functionality for the `Clear` button.